addons/web/static/src/env.js

```diff
@@ -134,13 +134,17 @@ async function _startServices(env, toStart, timeoutId) {
     if (toStart.size) {
         const names = [...toStart].map((s) => s.name);
         const missingDeps = new Set();
-        [...toStart].forEach((s) => s.dependencies.forEach((dep) => {
-            if (!(dep in services) && !names.includes(dep)) {
-                missingDeps.add(dep);
-            }
-        }));
+        [...toStart].forEach((s) =>
```

Comment:
Should we have those parentheses around the `s`?

Reply:
since the only change in the config was the `(s) =>` to `s =>` and that most of the team didn't care, we dropped it as well. So the linting is very small.